deploy/K8s-onprem-MIG/flower-service-nginx.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: flower-nginx
+  labels:
+    app: flower
```

Comment:
So this is the flower demo? Isn't there a client application that is needed to drive the entire demo?